Sources/Metric Logs/MetricViews.swift

```diff
@@ -0,0 +1,297 @@
+//
+//  MetricViews.swift
+//  DebugKit
+//  
+//  Copyright © 2022 Bottle Rocket Studios. All rights reserved.
+//
+
+import MetricKit
+import SwiftUI
+
+protocol MetricConfigurableView: View {
+    associatedtype Metric
+    init(payload: MetricPayload, metric: Metric)
+}
+
+// MARK: - MetricView
+struct MetricView<T: Foundation.Unit>: View {
+
+    // MARK: - Kind Subtype
+    enum Kind {
+        case histogram(Histogram<T>, formatter: MeasurementFormatter = .naturalScale)
+        case measurement(Measurement<T>, formatter: MeasurementFormatter = .naturalScale)
+        case average(MXAverage<T>, formatter: MeasurementFormatter = .naturalScale)
+        case value(Int)
+    }
+
+    // MARK: - Properties
+    let title: String
+    let kind: Kind
+
+    // MARK: - View
+    var body: some View {
+        Group {
+            switch kind {
+            case let .histogram(histogram, formatter): HistogramView.displaying(histogram: histogram, withTitle: title, using: formatter)
+            case let .measurement(measurement, formatter): MeasurementLabel.displaying(measurement: measurement, withName: title, using: formatter)
+            case let .average(average, formatter): AverageLabel.displaying(average: average, using: formatter, withTitle: title)
+            case .value(let value): MeasurementLabel.displaying(value: value, withName: title)
+            }
+        }
+    }
+}
+
+struct AnimationMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXAnimationMetric
+
+    // MARK: - View
+    var body: some View {
+        List {
+            MetricView(title: "Hitch Time Ratio", kind: .measurement(metric.scrollHitchTimeRatio))
+        }
+        .navigationTitle("Animation")
+    }
+}
+
+struct CellularMetricsView: MetricConfigurableView {
+
+    // MARK: - Properties
+    let payload: MetricPayload
+    let metric: MXCellularConditionMetric
+
+    // MARK: - View
+    var body: some View {
+        if let cellularConditions = payload.cellularConditions {
+            MetricView(title: "Cellular Conditions", kind: .histogram(cellularConditions, formatter: .providedUnit))
+        } else {
+            NoEntriesView(configuration: .default)
+                .navigationTitle("Cellular Conditions")
```

Comment:
The navigation title should be moved out of the else like `DisplayMetricsView` does.

Reply:
Moved it into an enclosing `Group`